CorsixTH/Lua/languages/english.lua

```diff
@@ -640,7 +675,7 @@ errors = {
   map_file_missing = "Could not find the map file %s for this level!",
   minimum_screen_size = "Please enter a screen size of at least 640x480.",
   unavailable_screen_size = "The screen size you requested is not available in fullscreen mode.",
-  alien_dna = "NOTE: There are no animations for Alien patients for sitting down, opening or knocking on doors etc. So, like with Theme Hospital to do these things they will appear to change to normal looking and then change back.  Patients with Alien DNA will only appear if they are set to in the level file",
+  alien_dna = "NOTE: There are no animations for Alien patients for sitting down, opening, or knocking on doors etc. So like with Theme Hospital, performing those actions will change the patient to normal then back to Alien.  Patients with Alien DNA will only appear if they are set to in the level file.",
```

Comment:
New comma is debatable. We want to convey sitting (on benches), opening (doors), or knocking on doors; so the 2nd and 3rd terms are related to the following object but the first is not.

`So` can be dropped.